components/reusable-components/navigation/navigation.module.scss

```diff
@@ -27,34 +27,28 @@
   margin-left: 0.55em;
```

Comment:
remove this margin, so the spacing from left and right of the navigation are the same